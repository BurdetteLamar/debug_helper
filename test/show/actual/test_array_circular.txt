---
Array (size=1 name=test_array_circular):
  Element 0:
    Array (size=1):
      Element 0: "[[[...]]] (Array)"
