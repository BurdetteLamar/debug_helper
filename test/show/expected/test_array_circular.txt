---
Array (message='test_array_circular'):
  size: 1
  Element 0:
    Array:
      size: 1
      Element 0: Array [[[...]]]
